docs/unit-test-mocks.md

```diff
@@ -0,0 +1,172 @@
+# Unit Test Mocks
+
+Relationship between mockable functions, mock groups and source files:
```

Comment:
Oxford comma is for cool people. I see you aren't cool :)

Reply:
I'm too old for cool.

Reply:
Woot, I'm in!  I didn't know the name, but it's the only way to roll IMO!

Reply:
**Unless you’re [writing](https://www.grammarly.com/writing) for a particular publication or drafting an essay for school, whether or not you use the Oxford comma is generally up to you. However, omitting it can sometimes cause some strange misunderstandings.

I love my parents, Lady Gaga and Humpty Dumpty.**

QED.